src/components/Nav/NewCollective.vue

```diff
@@ -33,51 +33,60 @@
 			</NcActionButton>
 		</template>
 	</NcAppNavigationItem>
-	<div v-else class="collective-create">
-		<form v-show="editing" @submit.prevent.stop="createCollective">
-			<NcEmojiPicker :show-preview="true" @select="addEmoji">
-				<NcButton type="tertiary"
-					:aria-label="t('collectives', 'Select emoji for collective')"
-					:title="t('collectives', 'Select emoji')"
-					class="button-emoji"
-					@click.prevent>
-					{{ emoji }}
-				</NcButton>
-			</NcEmojiPicker>
-
-			<input v-if="!pickCircle"
-				ref="nameField"
-				v-model="text"
-				:placeholder="t('collectives', 'New collective name')"
-				type="text"
-				required>
-			<NcMultiselect v-else
-				ref="circleSelector"
+	<div v-else v-show="editing" class="collective-create">
+		<NcEmojiPicker :show-preview="true" @select="addEmoji">
+			<NcButton type="tertiary"
+				:aria-label="t('collectives', 'Select emoji for collective')"
+				:title="t('collectives', 'Select emoji')"
+				class="button-emoji"
+				@click.prevent>
+				{{ emoji }}
+			</NcButton>
+		</NcEmojiPicker>
+
+		<NcTextField v-if="!pickCircle"
+			:value.sync="text"
+			:error="!text"
+			:label="t('collectives', 'New collective name')"
+			:show-trailing-button="!!text"
+			required
+			trailing-button-icon="arrowRight"
+			:trailing-button-label="t('collectives', 'Create new collective')"
+			@trailing-button-click="createCollective" />
+
+		<div v-else class="circle-selector--container">
+			<NcSelect ref="circleSelector"
```

Comment:
Not sure whether it's the switch from `NcMultiSelect` to `NcSelect`, but when selecting a circle with a longer name, the select field size grows larger than it should. Also the cancel and expand/collapse buttons inside the select field are not properly aligned.

Reply:
I can not confirm this, for me (Firefox 109, collectives on current NC master) it works as expected and looks like this:
![image](https://user-images.githubusercontent.com/1855448/218491881-3029cfd3-ce7b-4d73-82bc-ef120a891a6f.png)

Reply:
Ah, and it only grows bigger when closing the dropdown. With the dropdown open it keeps its desired size, like in your screenshot.

Reply:
Also, for some reason the selection is dark even when I use the light theme :thinking:

Reply:
> Also, for some reason the selection is dark even when I use the light theme thinking

Nevermind this. It seems to be a bug in nextcloud-vue. At least all other NcSelect fields are dark as well for me when my browser prefers dark theme but Nextcloud is set to light theme.

Reply:
Yes this is known and already fixed, waiting for release: https://github.com/nextcloud/nextcloud-vue/pull/3707

Reply:
> Ah, and it only grows bigger when closing the dropdown. With the dropdown open it keeps its desired size, like in your screenshot.

I pushed a commit which fixes parts of this issue. The other  parts are fixed with: https://github.com/nextcloud/nextcloud-vue/pull/3759

> Nevermind this. It seems to be a bug in nextcloud-vue. At least all other NcSelect fields are dark as well for me when my browser prefers dark theme but Nextcloud is set to light theme.

Fixed if we upgrade `@nextcloud/vue` to the new released `7.6.0`.